### PR TITLE
Remove bootstrap 2 and 3

### PR DESCRIPTION
**Pull request by samuelhwilliams**
Removes Bootstrap2 and Bootstrap3 themes, both of which are now comfortably replaced by Bootstrap4 (and, really, 5 when we add it).

We want to support add support for Bootstrap5 soon. This will probably be easier without the thought of having to support anything from bs2 and bs3.

### CODE REVIEW

**Comment by hasansezertasan** [comment]
> We want to support add support for Bootstrap5 soon. This will probably be easier without the thought of having to support anything from bs2 and bs3.

Yes and no, X-Editable doesn't work with BS5 😞.

File: flask_admin/theme.py [hunk 1]
```diff
@@ -11,12 +11,10 @@ class Theme:
...
-    folder: typing.Literal['bootstrap2', 'bootstrap3', 'bootstrap4']
+    folder: typing.Literal['bootstrap4']
     base_template: str = 'admin/base.html'
     swatch: str = 'default'
     fluid: bool = False
 
 
-Bootstrap2Theme = partial(BootstrapTheme, folder='bootstrap2')
-Bootstrap3Theme = partial(BootstrapTheme, folder='bootstrap3')
 Bootstrap4Theme = partial(BootstrapTheme, folder='bootstrap4')
```
Comment (hasansezertasan) [inline flask_admin/theme.py:20]: What about declaring Bootstrap4Theme without partial?

```suggestion
class Bootstrap4Theme(BootstrapTheme):
    folder='bootstrap4'
```

**Review by hasansezertasan — APPROVED** [review]
Looks good to me, good job 👍💯👍.

File: flask_admin/theme.py (hunk 1 above)
Comment (samuelhwilliams) [reply flask_admin/theme.py:20]: I think, because `BootstrapTheme` is a dataclass, it's preferable to me to just instantiate it than subclass it :)

**Comment by samuelhwilliams** [comment]
Ugh, deleted my fork unintentionally 🙃  I will re-open and merge, referencing this.

**Pull request closed without merging**